src/components/DateField.jsx

```diff
@@ -5,29 +5,13 @@ import {
   MuiPickersUtilsProvider,
   KeyboardDatePicker,
 } from '@material-ui/pickers';
-import PropTypes from 'prop-types';
 
 function DateField(props) {
```

Comment:
Maybe `DatePicker` instead of `DateField` to help show the connection with the pickers themselves.